recipes/resiprocate/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+import shutil
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+
+class ResiprocateConan(ConanFile):
+    name = "resiprocate"
+    description = "The project is dedicated to maintaining a complete, correct, and commercially usable implementation of SIP and a few related protocols. "
+    topics = ("sip", "voip", "communication", "signaling")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.resiprocate.org"
+    license = "VSL-1.0"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"fPIC": [True, False],
+               "shared": [True, False],
+               "with_ssl": [True, False],
+               "with_postgresql": [True, False],
+               "with_mysql": [True, False]}
+    default_options = {"fPIC": True,
+                       "shared": False,
+                       "with_ssl": True,
+                       "with_postgresql": True,
+                       "with_mysql": True}
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        if self.settings.os in ("Windows", "Macos"):
+            raise ConanInvalidConfiguration("OS is not supported")
+        if self.options.with_ssl:
+            self.requires("openssl/1.1.1h")
+        if self.options.with_postgresql:
+            self.requires("libpq/11.5")
+        if self.options.with_mysql:
+            self.requires("libmysqlclient/8.0.17")
+
+    def source(self):
+        if not self.version:
+            self.version = next(iter(self.conan_data["sources"]))
+        tools.get(**self.conan_data["sources"][self.version])
+        shutil.rmtree(self._source_subfolder, ignore_errors=True)
```

Comment:
I dont think this needed in CCI

```suggestion
```

Reply:
It just make easier command `conan source .`